gtsfm/frontend/global_descriptor/global_descriptor_base.py

```diff
@@ -0,0 +1,17 @@
+"""Base class for global descriptor implementations.
+
+Authors: John Lambert
+"""
+
+import abc
+
+import numpy as np
+
+from gtsfm.common.image import Image
+
+
+class GlobalDescriptorBase:
```

Comment:
Missing documentation here, for the class and for the args of describe()

Reply:
thanks, updated